src/index.tsx

```diff
@@ -1,5 +1,8 @@
-import { createRoot } from 'react-dom/client';
+import ReactDOM from 'react-dom';
+
 import { App } from './App';
 
-createRoot(document.getElementById('root') as HTMLElement)
-  .render(<App />);
+ReactDOM.render(
```

Comment:
![image](https://github.com/mate-academy/react_dynamic-list-of-posts/assets/56752562/bdca958c-115c-4e81-a678-4ace45d69055)
Try to fix this linter error